test/unittest/gtest_helpers.hpp

```diff
@@ -252,6 +288,42 @@ inline testing::Matcher<const CanId&> SourceNodeOfCanIdEq(NodeId node_id)
     return SourceNodeCanIdMatcher(node_id);
 }
 
+class DestinationNodeCanIdMatcher
+{
+public:
+    using is_gtest_matcher = void;
+
+    explicit DestinationNodeCanIdMatcher(NodeId node_id)
+        : node_id_{node_id}
+    {
+    }
+
+    bool MatchAndExplain(const CanId& can_id, std::ostream* os) const
+    {
+        const auto node_id = (can_id >> 7) & CANARD_NODE_ID_MAX;
+        if (os)
+        {
+            *os << "node_id=" << node_id;
+        }
+        return node_id == node_id_;
+    }
+    void DescribeTo(std::ostream* os) const
+    {
+        *os << "node_id==" << node_id_;
+    }
+    void DescribeNegationTo(std::ostream* os) const
+    {
+        *os << "node_id!=" << node_id_;
+    }
+
+private:
+    const NodeId node_id_;
+};
+inline testing::Matcher<const CanId&> DestinationNodeOfCanIdEq(NodeId node_id)
+{
+    return DestinationNodeCanIdMatcher(node_id);
+}
+
```

Comment:
All these gmock matchers (and `PrintTo` helpers) have no use when all the tests are green (b/c there is no output), but it makes big difference when something goes wrong. Using them (in conjunction with `EXPECT_THAT` macro) gives you nice print of what was expected and what actually is. Stay tuned, even more will be added...